### DISCUSSION TITLE
Shop is ineligible for shipping functions

### DISCUSSION THREAD

**Discussion by Sam115200** [Shopify/function-examples#543] [Feedback: Discounts APIs]
When using the shipping discount function, it works on many stores without any issues. But for some stores, there is this error which prevents us from creating the shipping discount,

`Shop is ineligible for shipping functions`

Does anybody know, what does this mean and why is that store not eligible for using shipping discount function? How to get rid of this error?

Any help is much appreciated, thanks.

**Comment by gil--** [comment]
Is the store(s) on Shopify Plus? It could be they're on checkout.liquid which does not support Functions.

**Reply by Sam115200** [reply]
2 stores had this issue so far, one of them is on Shopify plus and the other is on Shopify plan. I can understand perhaps checkout extensibility is not enabled on the plus store (I couldn't confirm this), but I can't figure out why the Shopify plan store had this issue.

**Reply by jharlap** [reply]
Hi Sam! Reach out to me on Shopify Partners Slack and I can ask for some specific details to take a closer look. I'm "Jonathan Harlap" there ;)

**Reply by jharlap** [reply]
For anyone else that finds this, the issue was that the shop had not updated to checkout extensibility and/or still used checkout.liquid. Product and Order Discount Functions may be used on a shop that still uses checkout.liquid, but Shipping Discount Functions can not.
